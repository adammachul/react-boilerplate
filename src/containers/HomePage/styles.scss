.HomePage {
    color: #D31F10;
}
