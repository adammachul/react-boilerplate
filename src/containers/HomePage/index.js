import React from 'react';

import styles from './styles.scss';

const HomePage = () => (
    <div className={styles.HomePage}>
        <h1> Home Pagee </h1>
    </div>
)

export default HomePage;
