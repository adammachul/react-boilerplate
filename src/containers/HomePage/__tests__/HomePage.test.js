import React from 'react';
import renderer from 'react-test-renderer';

import HomePage from '../index';

describe('<HomePage />', () => {
    test('renders', () => {
        const tree = renderer.create(<HomePage />).toJSON();

        expect(tree).toMatchSnapshot();
    })
})
