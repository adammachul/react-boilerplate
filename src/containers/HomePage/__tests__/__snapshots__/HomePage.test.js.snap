// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> renders 1`] = `
<div>
  <h1>
     Home Pageeee 
  </h1>
</div>
`;
